api/v1alpha1/subnet_types.go

```diff
@@ -28,10 +28,12 @@ type SubnetSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
 
-	Ipam      string   `json:"Ipam"`
-	Addresses []IpPool `json:"Addresses"`
-	Routes    []string `json:"Routes,omitempty"`
-	DNS       string   `json:"DNS,omitempty"`
+	Ipam           string   `json:"Ipam"`
```

Comment:
Here we should put an Enum kubebuilder comments for validation purposes. We should allow only the whereabouts values to be set a nothing else. Like this: https://github.com/Nordix/eno/blob/master/api/v1alpha1/l2serviceattachment_types.go#L38

Reply:
ok will do it now and push the changes.